bio/dada2/sample-inference/meta.yaml

```
@@ -0,0 +1,13 @@
+name: dada2_learn_errors
```

This is the same as for the `dada2/learn-errors` wrapper. Shouldn't this instead read something like:
```suggestion
name: dada2_sample_inference
```

---

Or simply `dada2_dada` instead. As `dada` is the function that is called by this wrapper.

---

Oups that was a typo, thanks for spotting it! As for the name, I preferred to name the rule as per the step name in the [tutorial](https://benjjneb.github.io/dada2/tutorial.html) rather than the function. In order to ease the readability for the (coming soon) meta-wrapper ;)

---

So `sample_inference` [works here](https://benjjneb.github.io/dada2/tutorial.html#sample-inference), right?

You might just want to double-check in the end. I changed the name of wrapper (6/10), that also had `dada2_learn_errors` to `dada2_merge_pairs` and you might want to go for `dada2_merge_paired_reads`, [according to the tutorial](https://benjjneb.github.io/dada2/tutorial.html#merge-paired-reads). So maybe something for the cleanup PR, as well (see below).